docs/fsharp/language-reference/members/methods.md

```diff
@@ -96,6 +100,18 @@ The following example illustrates a derived class that overrides a base class me
 ## Overloaded Methods
 Overloaded methods are methods that have identical names in a given type but that have different arguments. In F#, optional arguments are usually used instead of overloaded methods. However, overloaded methods are permitted in the language, provided that the arguments are in tuple form, not curried form.
 
+## Optional Arguments
+
+Starting with F# 4.1, you can also have C#-style optional arguments with a default parameter value in methods.  This is to help facilitate interoperation with C# code.  The following example demonstrates the syntax:
```

Comment:
I'd leave off the comparison to C#.  Other languages (notably C++, and Scala) also support optional arguments.